recipes-support/snapd/snapd_2.47.1.bb

```diff
@@ -43,7 +43,8 @@ RDEPENDS_${PN} += "		\
 S = "${WORKDIR}/${PN}-${PV}"
 
 EXTRA_OECONF += "			\
-	--disable-apparmor		\
+	--enable-apparmor		\
+	--enable-seccomp		\
```

Comment:
This is ok. I think the recipes for libseccomp should be copied over from meta-security, so that one is not required to use meta-security to be able to build snapd.

Reply:
mmh, I'd rather not copy stuff around, but simply require apparmor. requiring meta-seecurity is not a problem, but a feature. also automatically add any necessary kernel config. possibly hide this behind an option "full-confinement" in the recipe here? if that is possible. my general thinking would be: any code and options required for snapd should reside and be maintained only in 1 place - that is here. if the stuff required for full confinement is not here, every user will need to dig out the required bits and waste time ... eg I now am digging on the web what kernel options are required for full apparmor and how to tell yocto that. waste of time;)